CHANGELOG.txt

```diff
@@ -330,6 +330,9 @@
                 - "creme_core/bricks/generic/hat-bar.html"
         # Javascript :
             - Removed ChainedSelect.clone() method : not used and does not work anyway.
+            - Refactor creme.notification.NotificationBox & add unit tests
```

Comment:
It's more an "heavy rework" than a "refactor" ; OK to not list all breaking changes, but the message should indicates than there are many changes.
"add unit tests" is useless here.